README.md

```diff
@@ -316,7 +316,24 @@ Example：
     }
 }
 ```
+##### Degradation: Category=degradation
 
+[JSON Schema](https://github.com/cloudwego/kitex/blob/develop/pkg/#L30)
+
+|Variable|Introduction|
+|------------|----|
+| percentage | The percentage of dropped requests| 
+
+Example：
+default configuration(false,0)
+```json
+{
+   "degradation": {
+            "enabled": true,
+            "percentage": 0
```

Comment:
Setting percentage as 0 is kinda weird when degradation is enabled.
It's a good idea to make the documentation more uniform, see config-etcd, etc.